lib/common/xml.c

```diff
@@ -1762,16 +1757,13 @@ log_data_element(int log_level, const char *file, const char *function,
         prefix = "";
     }
 
-    /* Since we use the same file and line, to avoid confusing libqb, we need to use the same format strings */
     if (data == NULL) {
-        do_crm_log_alias(log_level, file, function, line, "%s: %s", prefix,
-                         "No data to dump as XML");
+        do_crm_log(log_level, "%s: %s", prefix, "No data to dump as XML");
```

Comment:
This format string doesn't make much sense to me if `prefix` is the empty string or contains only spaces.

Reply:
Yeah, I think we just never end up in this function with data==NULL. To make it even more fun, some callers of `crm_log_xml_*` use a ": " in the prefix, and some bracket the prefix.

The strange usage was added in 37161114 -- previously it was just a crm_warn() without the prefix. The comment about libqb no longer applies since we're not logging under an alias anymore.

I'm thinking we should drop the ": " here. We could be more consistent with the `crm_log_xml_*()` callers as far as brackets or whatnot, but that's not important now.